stack/docker-compose.yml

```diff
@@ -221,6 +221,44 @@ services:
     networks:
       - services
 
+  refarch-gateway:
+    image: ghcr.io/it-at-m/refarch/refarch-gateway:1.3.1@sha256:702d063d7141bb79af0107956b699c7771d0a91128f7622bdadc771911b94fca
+    depends_on:
+      wls-init-keycloak:
+        condition: service_completed_successfully
+    ports:
+      - "8083:8080"
+    environment:
+      - ALLOWED_ORIGINS_PUBLIC=http://localhost:*
+      - ALLOWED_ORIGINS_CLIENTS=http://localhost:*
+      - SPRING_CLOUD_GATEWAY_ROUTES_0_ID=sso
+      - SPRING_CLOUD_GATEWAY_ROUTES_0_URI=http://kubernetes.docker.internal:8100/
+      - SPRING_CLOUD_GATEWAY_ROUTES_0_PREDICATES_0=Path=/api/sso/userinfo
+      - SPRING_CLOUD_GATEWAY_ROUTES_0_FILTERS_0=RewritePath=/api/sso/userinfo, /auth/realms/${SSO_REALM}/protocol/openid-connect/userinfo
```

Comment:
```suggestion
      - SPRING_CLOUD_GATEWAY_ROUTES_0_FILTERS_0=RewritePath=/api/sso/userinfo, /${KEYCLOAK_RELATIV_PATH}/realms/${SSO_REALM}/protocol/openid-connect/userinfo
```

Reply:
in commit [d1482df](https://github.com/it-at-m/Wahllokalsystem/pull/582/commits/d1482df01727b1703ba72222f59e1a758b17654e) angepasst